lib/gems/pending/util/log_dump_obj.rb

```diff
@@ -0,0 +1,35 @@
+module LogDumpObj
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def dumpObj(obj, prefix = nil, prnt_obj = STDOUT, prnt_meth = :puts, options = {})
```

Comment:
why is `print` abbreviated to `prnt`?  I can *maybe* understand abbreviating `object` to `obj` and `method` to `meth`, but even there I am not sure what the benefit is.